src/main.rs

```diff
@@ -35,6 +35,8 @@ enum Cli {
     AddPerson { github_name: String },
     #[structopt(name = "static-api", help = "generate the static API")]
     StaticApi { dest: String },
+    #[structopt(name = "dump-person", help = "print information about a person")]
```

Comment:
let's not call it this, it sounds a bit dehumanizing

Reply:
I was planning on calling it `show-person`, but the equivalent for teams is called `dump-team`, so I thought it should be a similar name. I guess it's still different from `dump-team`, and I agree that it sounds weird :)

Reply:
Okay, I renamed the command to `show-person`, and I updated the commits. Unfortunately, I can't rename the branch without opening a new PR (hopefully GitHub will change that at some point).